holidays/countries/morocco.py

```diff
@@ -44,6 +44,11 @@ def _populate(self, year):
         # New Year's Day
         self._add_new_years_day("Nouvel an - Premier janvier")
 
+        # Amazigh New Year
```

Comment:
Better to place it after Independence Manifesto Day, to keep the chronological order.